data/campaigns/Sceptre_of_Fire/scenarios/5_Hills_of_the_Shorbear_Clan.cfg

```diff
@@ -544,15 +549,15 @@
         [/kill]
         [message]
             speaker=Durstorn
-            message= _ "Ha! Now we can get those tools easily, and go back to our own caves."
+            message= _ "Ha! Now we can easily take their tools back to our own caves."
         [/message]
         [message]
             speaker=Thursagan
             message= _ "And how do you plan on doing that? There are elves swarming these hills, trying to kill us."
         [/message]
         [message]
             speaker=Durstorn
-            message= _ "You are right... well, we will be able to break out eventually, and while we’re here, we’ll be able to finally make the sceptre."
+            message= _ "You are right... well, we can close up these caves and make the sceptre in here. I’m sure we’ll eventually come up with a plan for breaking out of here when we’re done."
```

Comment:
> @nemaara please could you weigh in on this.

Anyone else we could get to weigh in here if she can't? `git blame` seems kind of confused by people fixing apostrophes, which doesn't really count as "writing" the line previously...

Reply:
You can use blame on older revisions. Judging from https://github.com/wesnoth/wesnoth/blame/master/data/campaigns/Sceptre_of_Fire/scenarios/5_Hills_of_the_Shorbear_Clan.cfg, the phrase seems to be 11 years old.

I haven't really followed this particular conversation but my reading of the original line is that 'break out' refers to the siege from the elves. I'm not sure what the most recent proposal (from 12th October) is supposed to add - if anything it looks like it takes away from the original text. Given a choice between that proposal and the original, I say leave the original - but that's just my opinion.

Reply:
My issue with the original text is that Thursagan is asking "How do you plan on doing that?"
But Durstorn's response does not sound like a plan of action. Instead, it sounds like he is describing a circumstance. I want to make Durstorn's response sound like a definitive plan of action.

Reply:
It can be worded better but I get your idea. You could say something like they'll barricade themselves in and then they'll make a plan to break the siege later. If you're more specific about the subject of their actions it'll probably read better in this instance.

Reply:
How does this sound:
> Er, yes I know... well, what if we buy ourselves some time? We can close the gates to keep the elves out until we have a good plan. We can even make the sceptre while we're here.

Reply:
Durstorn should sound more demanding/directive to me. So something more like "Aye. So close the gates and stay them elves outside. Meanwhiles ye runecrafter ken craft tha sceptre and I'll think me a plan ta get out o' here."

You can make it more in plain english of course, the idea is just to make him sound like he's in charge. Durstorn ain't diplomatic, I think.

(Also you have two comments left below to address too haha).

Reply:
Regarding the scottish english, ill review durstorn's lines again to see if it would be more consistent to make this more scottish sounding or more plain sounding.

Reply:
I reviewed Durstorn's lines and can confirm that he talks in plain English, not in Scottish English.
Is this line good @nemaara :
> "Yes, I'm aware! Just close the gates and keep those elves out! We'll think of something. As long as a dwarf is in a cave, he can come up with a plan. And since the Shorbear's rune workshops are in here, you and your runecrafters might as well make the Sceptre now."

Reply:
> I'm aware! Close the gates and keep them out! It should be easy to come up with a plan to outwit them later. Meanwhile, since the Shorbear's rune workshops are in here, you and your runecrafters might as well make that Sceptre now.